contrib/babelfishpg_common/src/numeric.c

```diff
@@ -1029,3 +1042,48 @@ tsql_numeric_get_typmod(Numeric num)
 
 	return (((precision & 0xFFFF) << 16 ) | (scale & 0xFFFF)) + VARHDRSZ;
 }
+
+Datum
+bigint_sum(PG_FUNCTION_ARGS)
```

Comment:
As discussed offline, we should implement this final function to have support for both platforms, one which supports 128bit intger and one which does not support (similar to what postgres provides)

Reply:
Resolved in the latest commit